rclcpp/src/rclcpp/callback_group.cpp

```diff
@@ -14,17 +14,20 @@
 
 #include "rclcpp/callback_group.hpp"
 
+#include <memory>
 #include <vector>
 
 using rclcpp::CallbackGroup;
 using rclcpp::CallbackGroupType;
 
 CallbackGroup::CallbackGroup(
+  rclcpp::Context::SharedPtr context_ptr,
```

Comment:
Specifically this change is breaking API.

Reply:
removed it because of deferring to create the guard condition in the callback gruop